### PR TITLE
Adding route for liveness and readiness probes.

### PR DESCRIPTION
**Pull request by omaciel**
# Description

This is required for AppSRE.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by acosferreira — DISMISSED** [review]
lgtm

**Review by loadtheaccumulator — DISMISSED** [review]

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -41,23 +41,21 @@ objects:
         livenessProbe:
           failureThreshold: 3
           httpGet:
-            path: /
-            port: 9000
+            path: ${READINESS_URI}
+            port: 8080
```
Comment (psav) [inline deploy/clowdapp.yaml:45]: I'd set the port to `web`, 8080 is not right for clowder, and `web` will use whatever clowder picked

**Review by psav — CHANGES_REQUESTED** [review]

**Review by ldjebran — APPROVED** [review]
/LGTM

**Review by ldjebran — DISMISSED** [review]

**Pull request closed without merging**